test/validate/linux_validate.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	cnsLabelSelector        = "k8s-app=azure-cns"
+	validatorPod            = "k8s-app=azure-cns"
```

Comment:
I needed to break any usages of it that the go language server wasn't aware of (due to platform build tags or things like that), because anywhere that it is used needs the container name passed also.

Reply:
And it may be a label selector, but it's misleading named. We're not really looking for the CNS Pod when we use this, we're looking for a Pod that we can exec in to to interrogate the CNS statefiles. This **can't** be the CNS Pod anymore because CNS doesn't have a shell.*
*I added the debug container to the CNS Pod as a hack to get these tests passing again. But the test should never have relied on internal implementation details of the CNS Pod like that it may contain a shell. The tests should have spun up a debug pod to exec to and interrogated the statefile from there, relying purely on external observable behavior.